OrbitClientData/FunctionUtils.cpp

```diff
@@ -2,13 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "FunctionUtils.h"
+#include "OrbitClientData/FunctionUtils.h"
 
 #include <map>
 
 #include "OrbitBase/Logging.h"
-#include "Path.h"
-#include "Utils.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/strings/match.h"
+#include "xxhash.h"
+
+namespace {
+uint64_t StringHash(const std::string& string) {
```

Comment:
I wonder if the Hash is still needed. It seems to be used in presets and pdb, but from the looks of it this can be dropped in favor of name.

Reply:
I am somewhat sure is only used in presets. I would also be in favor to use the function name instead (actually probably demangled name)
I tried to keep this PR as small as possible (haha) (but actually). Since changing the presets requires multiple changes in a few places, I will refer this to a different PR for now.